wolfcrypt/benchmark/benchmark.c

```diff
@@ -483,6 +486,14 @@
 #define BENCH_DILITHIUM_AES_LEVEL3_SIGN 0x40000000
 #define BENCH_DILITHIUM_AES_LEVEL5_SIGN 0x80000000
 
+/* Post-Quantum Asymmetric algorithms. (Part 2) */
```

Comment:
Can you please fix the `./wolfcrypt/benchmark/benchmark -?` to show the `-pq` options:

```
#ifndef WOLFSSL_BENCHMARK_ALL
    printf("%s", bench_Usage_msg1[lng_index][13]);   /* option -<alg> */
    printf("             ");
    line = 13;
    for (i=0; bench_cipher_opt[i].str != NULL; i++)
        print_alg(bench_cipher_opt[i].str + 1, &line);
    printf("\n             ");
    line = 13;
    for (i=0; bench_digest_opt[i].str != NULL; i++)
        print_alg(bench_digest_opt[i].str + 1, &line);
    printf("\n             ");
    line = 13;
    for (i=0; bench_mac_opt[i].str != NULL; i++)
        print_alg(bench_mac_opt[i].str + 1, &line);
    printf("\n             ");
    line = 13;
    for (i=0; bench_asym_opt[i].str != NULL; i++)
        print_alg(bench_asym_opt[i].str + 1, &line);
    printf("\n             ");
    line = 13;
    for (i=0; bench_other_opt[i].str != NULL; i++)
        print_alg(bench_other_opt[i].str + 1, &line);
    printf("\n");
#endif
```

```
./wolfcrypt/benchmark/benchmark -?
...
-<alg>      Algorithm to benchmark. Available algorithms include:
              cipher aes-cbc aes-gcm chacha20 chacha20-poly1305
              digest md5 poly1305 sha sha2 sha224 sha256 sha384 sha512 sha3
              sha3-224 sha3-256 sha3-384 sha3-512
              mac hmac hmac-md5 hmac-sha hmac-sha224 hmac-sha256 hmac-sha384
              hmac-sha512 pbkdf2
              asym rsa rsa-sz dh ecc-kg ecc ecc-all
              other rng
```

```
 % ./wolfcrypt/benchmark/benchmark -pq
------------------------------------------------------------------------------
 wolfSSL version 5.5.0
------------------------------------------------------------------------------
wolfCrypt Benchmark (block bytes 1048576, min 1.0 sec each)
kyber_level1-kg     136733 ops took 1.000 sec, avg 0.007 ms, 136733.000 ops/sec
kyber_level1-ed      57100 ops took 1.001 sec, avg 0.018 ms, 57020.797 ops/sec
kyber_level3-kg      85613 ops took 1.000 sec, avg 0.012 ms, 85612.306 ops/sec
kyber_level3-ed      38900 ops took 1.002 sec, avg 0.026 ms, 38816.542 ops/sec
kyber_level5-kg      63892 ops took 1.000 sec, avg 0.016 ms, 63891.924 ops/sec
kyber_level5-ed      28700 ops took 1.001 sec, avg 0.035 ms, 28666.742 ops/sec
kyber90s_level1-kg  165351 ops took 1.000 sec, avg 0.006 ms, 165351.000 ops/sec
kyber90s_level1-ed   66000 ops took 1.001 sec, avg 0.015 ms, 65958.704 ops/sec
kyber90s_level3-kg  120247 ops took 1.000 sec, avg 0.008 ms, 120246.283 ops/sec
kyber90s_level3-ed   49100 ops took 1.001 sec, avg 0.020 ms, 49073.897 ops/sec
kyber90s_level5-kg   86336 ops took 1.000 sec, avg 0.012 ms, 86335.918 ops/sec
kyber90s_level5-ed   35900 ops took 1.000 sec, avg 0.028 ms, 35897.056 ops/sec
saber_level1-kg      93008 ops took 1.000 sec, avg 0.011 ms, 93007.534 ops/sec
saber_level1-ed      39300 ops took 1.000 sec, avg 0.025 ms, 39281.775 ops/sec
saber_level3-kg      56989 ops took 1.000 sec, avg 0.018 ms, 56988.429 ops/sec
saber_level3-ed      22000 ops took 1.002 sec, avg 0.046 ms, 21962.534 ops/sec
saber_level5-kg      33106 ops took 1.000 sec, avg 0.030 ms, 33105.211 ops/sec
saber_level5-ed      15600 ops took 1.005 sec, avg 0.064 ms, 15516.876 ops/sec
ntruHPS_level1-kg    20099 ops took 1.000 sec, avg 0.050 ms, 20099.000 ops/sec
ntruHPS_level1-ed    43200 ops took 1.002 sec, avg 0.023 ms, 43102.118 ops/sec
ntruHPS_level3-kg    11607 ops took 1.000 sec, avg 0.086 ms, 11606.189 ops/sec
ntruHPS_level3-ed    28100 ops took 1.003 sec, avg 0.036 ms, 28003.697 ops/sec
ntruHPS_level5-kg     9153 ops took 1.000 sec, avg 0.109 ms, 9152.642 ops/sec
ntruHPS_level5-ed    18800 ops took 1.003 sec, avg 0.053 ms, 18740.819 ops/sec
FALCON     1 sign         4500 ops took 1.002 sec, avg 0.223 ms, 4489.718 ops/sec
FALCON     1 verify      24900 ops took 1.001 sec, avg 0.040 ms, 24865.882 ops/sec
FALCON     5 sign         2400 ops took 1.017 sec, avg 0.424 ms, 2358.797 ops/sec
FALCON     5 verify      11900 ops took 1.000 sec, avg 0.084 ms, 11896.038 ops/sec
DILITHIUM     2 sign        15200 ops took 1.006 sec, avg 0.066 ms, 15109.358 ops/sec
DILITHIUM     2 verify      38800 ops took 1.001 sec, avg 0.026 ms, 38766.856 ops/sec
DILITHIUM     3 sign         9400 ops took 1.003 sec, avg 0.107 ms, 9368.989 ops/sec
DILITHIUM     3 verify      25700 ops took 1.003 sec, avg 0.039 ms, 25630.770 ops/sec
DILITHIUM     5 sign         8300 ops took 1.002 sec, avg 0.121 ms, 8287.179 ops/sec
DILITHIUM     5 verify      15700 ops took 1.000 sec, avg 0.064 ms, 15695.902 ops/sec
DILITHIUM-AES     2 sign        22300 ops took 1.004 sec, avg 0.045 ms, 22203.458 ops/sec
DILITHIUM-AES     2 verify      60800 ops took 1.001 sec, avg 0.016 ms, 60746.485 ops/sec
DILITHIUM-AES     3 sign        15500 ops took 1.000 sec, avg 0.065 ms, 15494.455 ops/sec
DILITHIUM-AES     3 verify      44500 ops took 1.000 sec, avg 0.022 ms, 44483.137 ops/sec
DILITHIUM-AES     5 sign        11900 ops took 1.002 sec, avg 0.084 ms, 11881.892 ops/sec
DILITHIUM-AES     5 verify      30100 ops took 1.005 sec, avg 0.033 ms, 29951.652 ops/sec
Benchmark complete
```